.github/workflows/docker_images_template.yaml

```diff
@@ -24,14 +24,16 @@ jobs:
           head_ref: ${{ github.head_ref }}
 
       - name: Build image
+        env:
+          PYPI_TAG: ${{steps.get-ref.outputs.tag}}
         run: |
-          svn checkout https://github.com/dmwm/CMSKubernetes/trunk/docker/pypi/${{inputs.wmcore_component}}
-          cd ${{inputs.wmcore_component}}
-          sed -i -e "s,ENV TAG=.*,ENV TAG=${{steps.get-ref.outputs.tag}},g" Dockerfile
+          echo "Building service: ${{inputs.wmcore_component}}, with tag: ${PYPI_TAG}"
+          svn checkout https://github.com/dmwm/CMSKubernetes/trunk/docker/pypi/$SERVICE
+          cd $SERVICE
```

Comment:
@amaltaro is this variable $SERVICE exported from somewhere up?

Reply:
I meant to fix it yesterday evening. Sorry for the ill review request! Updating it in a second.